user_guide_src/source/outgoing/localization.rst

```diff
@@ -254,3 +254,45 @@ project:
 
 The translated messages will be automatically picked
 up because the translations folders get mapped appropriately.
+
+.. _generating-translation-files-via-command:
+
+Generating Translation Files via Command
+========================================
+
+.. versionadded:: 4.5.0
+
+You can automatically generate and update translation files in your **app** folder. The command will search for the use of the ``lang()`` function, combine the current translation keys in **app/Language** by defining the locale ``defaultLocale`` from ``Config\App``.
+After the operation, you need to translate the language keys yourself.
+The command is able to recognize nested keys normally ``File.array.nested.text``.
+Previously saved keys do not change.
+
+.. code-block:: console
+
+    php spark lang:find
+
+.. literalinclude:: localization/019.php
+
+The language files generated will most likely not conform to your coding standards.
+It is recommended to format them. For example, run ``vendor/bin/php-cs-fixer fix ./app/Language`` if ``php-cs-fixer`` is installed.
+
+.. note:: The command ignores **app/Language/** path.
```

Comment:
This note is a bit difficult to understand.
What is the command? What do you mean by ignores?

Reply:
This means: When scanning folders while searching for translations, **app/Language** will be skipped.

Reply:
Move the note right after the sample code?
and change the sentence to "When the command scans folders, app/Language will be skipped."?

Or is the note really needed?

Reply:
@neznaika0 Please take care of the above my comment.